lib/Doctrine/Sniffs/Spacing/ControlStructureSniff.php

```diff
@@ -57,7 +40,11 @@ public function register()
         ];
     }
 
-    public function process(File $phpcsFile, $stackPtr)
+    /**
+     * @phpcsSuppress SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint
```

Comment:
@Majkl578 I usually add a comment explaining, why the sniff is suppressed

Reply:
True, but I think the name is pretty self-explaining in this case: _Type Hint Declaration  -Missing Parameter Type Hint_